src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -670,8 +671,6 @@ def job_is_running(jobid):
     Returns True if job shows a running state and substate
     """
     jobinfo = printjob_info(jobid)
-    if 'state' in jobinfo and jobinfo['state'] != 'R':
-        return False
```

Comment:
Should the values in job_is_suspended() method also be updated? I see that it is also using printjob_info()

Reply:
No, that was already using only the substate, which is always numerical. In fact, this pull request makes job_is_running consistent with the current job_is_suspended routine.